docs/api-overview.mdx

```diff
@@ -115,7 +122,125 @@ The same ApiKey necessary to call /hasMatchingCredentials grants you and only yo
 
 The `url` from the response ought to then be enriched with a `redirectUrl` query param that will take the next step in your target service's user onboarding webpage. _More information on this can be found in the [Acceptance Guide](/acceptance-guide)._
 
----
+:::info Example Credential Request
+
+So for example, if you want to check if a user has credentials of type `SsnCredential`, `FullNameCredential` and `AddressCredential` issued by any issuer you would send the following request:
+
+<details>
+<summary>Request Body</summary>
+<p>
+
+```json
+{
+  "email": "richard@piedpiper.net",
+  "credentialRequests": [
+    {
+      "type": "SsnCredential",
+      "issuers": [],
+      "mandatory": "yes"
+    },
+    {
+      "type": "FullNameCredential",
+      "issuers": [],
+      "mandatory": "yes"
+    },
+    {
+      "type": "AddressCredential",
+      "issuers": [],
+      "children": [
+        {
+          "type": "CityCredential",
+          "issuers": [],
+          "mandatory": "if_available"
+        },
+        {
+          "type": "StateCredential",
+          "issuers": [],
+          "mandatory": "yes"
+        },
+        {
+          "type": "CountryCredential",
+          "issuers": [],
+          "mandatory": "yes"
+        },
+        {
+          "type": "ZipCodeCredential",
+          "issuers": []
+        }
+      ]
+    }
+  ]
+}
+```
+
+</p>
+</details>
+
+In the given example, the user will be asked to share their `SsnCredential`, `FullNameCredential` and `AddressCredential` credentials.
+
+Let's breakdown to understand what is happening here:
+
+- `SsnCredential` is a "[single attribute atomic credential](/schema#schema-definitions)" and therefore does not have any sub-credentials. Because it is marked as `mandatory`, it only matches if the user has a credential.
+
+```json title="SnnCredential"
+{
+  "type": "SsnCredential",
+  "issuers": [],
+  "mandatory": "yes"
+}
+```
+
+- `FullNameCredential` is a "[composite credential](/schema#schema-definitions)" and therefore has *2 sub-credentials*¹: `FirstNameCredential`, `LastNameCredential`. Because it is marked as `mandatory`, and it has no `children` attribute, it only matches if the user has a credential with all sub-credentials.
+
+```json title="FullNameCredential"
+{
+  "type": "FullNameCredential",
+  "issuers": [],
+  "mandatory": "yes"
+}
+```
+
+> _¹The `FullNameCredential` also has a `MiddleNameCredential`, but it's `optional`._
```

Comment:
Hmm this is confusing even for me! Are we saying that FirstName and LastName are mandatory but MiddleName is not if the FullName composite is? 

Shouldn't MIddleName be mandatory just like First and Last?

Reply:
I agree with Jacob

Reply:
But why then is FirstName and LastName implicitly considered manadatory but MiddleName is not? Where is this defined? 

I don't like it because how is one suppose to know what is mandatory by default in a composite cred and not? I think all the atomic creds in a composite cred should be considered mandatory or not mandatory.

Reply:
It definitely needs to be clear (at least in documentation, ideally encoded in the schema somehow) which child credentials are mandatory and which are not.

Reply:
I thought the `Schema Library` section was for that

https://github.com/UnumID/UnumID.github.io/pull/111#discussion_r1295311046